pyprland/plugins/scratchpads/__init__.py

```diff
@@ -539,15 +539,6 @@ async def run_show(self, uid: str) -> None:
             self.log.error("Failed to show %s, aborting.", uid)
             return
 
-        excluded_ids = scratch.conf.get("excludes", [])
-        if excluded_ids == "*":
```

Comment:
why did this black have to move?

Reply:
I think it was by mistake, I will put it to the same place.